test/index.test.js

```diff
@@ -110,7 +125,7 @@ describe('plugin', () => {
   }]);
 
   beforeEach(() => {
-    return redisClient.flushdb();
```

Comment:
Yeah the reason for the `return` would be if `flushdb` returned a `Promise`, and was therefore an asynchronous operation. When you return a `Promise` in a `beforeEach` it indicates to `Mocha` that it needs to wait for that `Promise` to resolve/reject before going ahead and running tests.

If it isn't asynchronous, though, and it blocks while flushing redis then we wouldn't need `return` because by the time the function actually returns then the flush will be complete and tests can proceed without issue.

I just checked and it looks like `flushdb` returns a `Promise`, though. Meaning we should still `return` it in the `beforeEach`

```js
=> console.log(redisClient.flushdb());
<= Promise {
    _bitField: 0,
    _fulfillmentHandler0: undefined,
    _rejectionHandler0: undefined,
    _progressHandler0: undefined,
    _promise0: undefined,
    _receiver0: undefined,
    _settledValue: undefined
  }
```